docs/security/access-control.md

```diff
@@ -46,6 +46,21 @@ Gravitino doesn't support metadata authentication. It means that Gravitino won't
 
 ### Authorization
 
+#### Build-in Authorization
+
+Gravitino provides built-in metadata authorization. You can enable the built-in metadata authorization through the following configuration.
+```
+gravitino.authorization.enable = true
+gravitino.authorization.impl = org.apache.gravitino.server.authorization.jcasbin.JcasbinAuthorizer
```

Comment:
Could we make `org.apache.gravitino.server.authorization.jcasbin.JcasbinAuthorizer` as the default implementation?

Reply:
fixed